src/Element/Element.tsx

```diff
@@ -197,7 +197,6 @@ const Element: React.FC<ElementProps> = ({
 
   const elementStyle: React.CSSProperties = useMemo(() => {
```

Comment:
I think we can completely remove this useMemo hook (it's useless now) and pass ELEMENT_STYLE directly to div.style

Reply:
Right. Let me fix that